zad001.rb

```diff
@@ -0,0 +1,67 @@
+if __FILE__ == $0
+
+  # Tresc zadania:
+  # In the above example, the sum of each three-measurement window is as follows:
+
+# A: 607 (N/A - no previous sum)
+# B: 618 (increased)
+# C: 618 (no change)
+# D: 617 (decreased)
+# E: 647 (increased)
+# F: 716 (increased)
+# G: 769 (increased)
+# H: 792 (increased)
+
+# In this example, there are 5 sums that are larger than the previous sum.
+
+# Consider sums of a three-measurement sliding window. How many sums are larger than the previous sum?
+# Koniec tresci. -----------------------------------------------------------------------------------
+
+
+  puts (' Type filename with extension .txt (or type EXIT if you want to quit) end press ENTER :')
```

Comment:
Usually people do not put a space between a method name and the parenthesis:

```suggestion
  puts(' Type filename with extension .txt (or type EXIT if you want to quit) end press ENTER :')
```

The same applies to other lines with `puts` in this file.`